lib/iris/experimental/um.py

```diff
@@ -685,19 +774,43 @@ def normalise(values, kind):
             output_file.seek((header.data_start - 1) * self._word_size)
             dataset_type = self.fixed_length_header.dataset_type
             sector_size = self._WORDS_PER_SECTOR * self._word_size
+
             for field in self.fields:
-                data = field.get_data()
-                if data is not None:
+                if isinstance(field, (Field2, Field3)):
```

Comment:
Not the biggest fan of using isinstance here. The old style was preferable IMHO - is that incompatible with these changes?